src/components/Footer/FooterNav/FooterNav.stories.tsx

```diff
@@ -4,10 +4,10 @@ import React from 'react'
 import { FooterNav } from './FooterNav'
 
 export default {
-  title: 'FooterNav',
+  title: 'Footer/FooterNav',
   parameters: {
     info: `
-    Used in USWDS 2.0 Footer component
+    Nav used in USWDS 2.0 Footer component. Can display simple nav items in a basic nav, or grouped nav items in an extended nav.
```

Comment:
I think we should try and be consistent with our descriptions and either stick with saying what component it is used in first or the actual description of the component. The order of the footer and address component descriptions are not consistent. This is definitely just a nit. It's not really a big deal.